### PR TITLE
feat(dh): load actors and add filters to Actors page

### PR DESCRIPTION
**Pull request by dzhavat**
### DataHub
- load actors in Actors page
- add "Status" and "Market roles" filters (non-functional for now)
- mock get actors query

## References

- https://github.com/Energinet-DataHub/team-titans/issues/215

### CODE REVIEW

File: libs/dh/market-participant/actors/feature-actors/src/lib/dh-actors-overview.component.ts [hunk 1]
```diff
@@ -38,15 +40,42 @@ interface Actor {
...
     status: { accessor: 'status' },
   };
+
+  filters$ = new BehaviorSubject<ActorsFilters>({
+    actorStatus: null,
+    marketRoles: null,
+  });
+
+  ngOnInit(): void {
+    this.getActorsQuery$.valueChanges.pipe(takeUntil(this.destroy$)).subscribe({
```
Comment (mimse) [inline libs/dh/market-participant/actors/feature-actors/src/lib/dh-actors-overview.component.ts:70]: Suggestion: Store the subscription reference of valueChanges, and call unsubscribe on that in ngOnDestory

File: libs/dh/market-participant/actors/feature-actors/src/lib/filters/dh-actors-filters.component.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  WattDropdownOption,
+  WattDropdownOptions,
+} from '@energinet-datahub/watt/dropdown';
+import { MarketParticipantEicFunction } from '@energinet-datahub/dh/shared/domain';
+import { ActorStatus } from '@energinet-datahub/dh/shared/domain/graphql';
+
+import { ActorsFilters } from '../actors-filters';
+
+/** Helper function for creating form control with `nonNullable` based on value. */
+const makeFormControl = <T>(value: T | null = null) =>
```
Comment (mimse) [inline libs/dh/market-participant/actors/feature-actors/src/lib/filters/dh-actors-filters.component.ts:45]: Question: I´m pretty sure a saw this in one of Bjørns PR´s as well. Maybe we should move it to a helper?

File: libs/dh/market-participant/actors/feature-actors/src/lib/filters/dh-actors-filters.component.ts [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  actorStatusOptions: WattDropdownOptions = this.buildActorStatusOptions();
+  marketRolesOptions = this.buildMarketRolesOptions();
+
+  ngOnInit() {
+    this.formGroup = new FormGroup({
+      actorStatus: makeFormControl<ActorStatus[] | null>(this.initial.actorStatus),
+      marketRole: makeFormControl<MarketParticipantEicFunction[] | null>(this.initial.marketRoles),
+    });
+
+    this.formGroup.valueChanges
```
Comment (mimse) [inline libs/dh/market-participant/actors/feature-actors/src/lib/filters/dh-actors-filters.component.ts:126]: Suggestion: Use the same pattern of saving the subscription ref here

**Review by mimse — CHANGES_REQUESTED** [review]

File: libs/dh/market-participant/actors/feature-actors/src/lib/filters/dh-actors-filters.component.ts (hunk 2 above)
Comment (dzhavat) [reply libs/dh/market-participant/actors/feature-actors/src/lib/filters/dh-actors-filters.component.ts:45]: Yes, it's the same helper function. Will move it to a shared lib in a follow up PR.

**Review by mimse — APPROVED** [review]

**Pull request merged**